src/form/Form.tsx

```diff
@@ -1345,7 +1336,15 @@ function Form({
       const email = fieldValues[fieldKey] as string;
       if (validators.email(email)) {
         clickPromise = setButtonLoader(button)
-          .then(() => sendMagicLink({ fieldVal: email }))
+          .then(() => {
+            if (getStytchJwt()) return sendMagicLink({ fieldVal: email });
```

Comment:
can we make a method that calls maybe !!getStytchJwt() and the method name is 

isAuthStytch? or something of that nature to be more clear